src/v3/src/components/TextWithHtml/TextWithHtml.tsx

```diff
@@ -57,6 +57,11 @@ const TextWithHtml: UISchemaElementComponent<{
   return (
     <Box>
       <Box
+        sx={(theme) => ({
+          [`& .${contentClassname}`]: {
+            color: theme.palette.primary.main,
+          },
+        })}
```

Comment:
This ensure the color of the link embedded in the translated string matches the rest of the application